backend/jupiterbrain.go

```diff
@@ -339,7 +367,12 @@ func (p *jupiterBrainProvider) Start(ctx gocontext.Context, startAttributes *Sta
 		}
 		instance.Stop(ctx)
 
-		return nil, ctx.Err()
+		select {
+		case err := <-errChan:
+			return nil, err
+		case <-time.After(2 * time.Second):
```

Comment:
This can possibly be reduced. `net.Dial` will return when the context is cancelled, so we don't need to wait for a full second. The HTTP call to Jupiter Brain might still be stalling, though.